handlers/RPCHelpers.cpp

```diff
@@ -41,6 +41,33 @@ deserializeTxPlusMeta(Backend::TransactionAndMetadata const& blobs)
     return result;
 }
 
+
+std::pair<
+    std::shared_ptr<ripple::STTx const>,
+    std::shared_ptr<ripple::TxMeta const>>
+deserializeTxPlusMeta(Backend::TransactionAndMetadata const& blobs, std::uint32_t seq)
+{
+    std::pair<
+        std::shared_ptr<ripple::STTx const>,
+        std::shared_ptr<ripple::TxMeta const>>
+        result;
+    {
+        ripple::SerialIter s{
+            blobs.transaction.data(), blobs.transaction.size()};
+        result.first = std::make_shared<ripple::STTx const>(s);
+    }
+    {
+        // ripple::Blob{blobs.metadata.data(), blobs.metadata.size()};
+
+        result.second =
+            std::make_shared<ripple::TxMeta const>(
+                result.first->getTransactionID(),
+                seq,
+                blobs.metadata);
+    }
+    return result;
+}
```

Comment:
Let's use the first `deserializeTxPlusMeta` function in the body of this one. `TxMeta` has a constructor that takes in an `STObject`, which is how the first function returns the metadata.